packaging/nnstreamer.spec

```diff
@@ -735,13 +734,6 @@ Provides additional gstreamer plugins for nnstreamer pipelines
 %define enable_tvm -Dtvm-support=disabled
 %endif
 
-# Support snpe
-%if 0%{?snpe_support}
-%define enable_snpe -Dsnpe-support=enabled
-%else
-%define enable_snpe -Dsnpe-support=disabled
-%endif
-
```

Comment:
Are you going to let "auto" decide snpe support?
(not a good idea for platform build because we need to ensure features for platform builds.)
Anyway, it's ok until the snpe-support packages are ready.

Reply:
Yes. I agreed with your opinion. It's not a good idea.
However, there is no Robot profile option in OBS so package config file (i.e. pc) for snpe-devel rpm is the only way to check whether SNPE is enabled or not.

As you mentioned above, it is a workaround so I will add the comment for this reason.